packages/lib/src/components/Card/Card.tsx

```diff
@@ -234,11 +225,7 @@ export class CardElement extends UIElement<CardElementProps> {
 
     render() {
         return (
-            <CoreProvider
-                i18n={this.props.i18n}
-                loadingContext={this.props.loadingContext}
-                commonProps={{ isCollatingErrors: this.props.SRConfig.collateErrors }}
-            >
+            <CoreProvider i18n={this.props.i18n} loadingContext={this.props.loadingContext}>
```

Comment:
I don't know how it is going to be when adding this feature to other payment methods and how much prop drilling will happen, but did you consider to create a provider/context exclusively for the screen reader mechanism? 

Ex:
```jsx
<SRPanelProvider srPanel={this.props.modules.srPanel} >
...
</SRPanelProvider>
```

Then, on the UI components, you could use a custom hook:

```jsx
const { srPanel, ...anyOtherHelperFunction } = useScreenReaderPanel();
```

The advantages I see is this can encapsulate any related screen reader logic in the hook, plus potentially avoid any prop drilling.

Reply:
I'm lost, how does it avoid prop drilling? 
Right now you can do the same, you can `useCoreProvider` to get the module.

Reply:
> I'm lost, how does it avoid prop drilling?

Same way that you end up using `useCoreProvider` to get the `i18n`. You can have similar approach to get the SRPanel and any other SR helper function.

> Right now you can do the same, you can useCoreProvider to get the module.

You definitely can, but I don't think we should keep adding more functionality to the main CoreProvider - otherwise in the future it will end up handling multiple non-related things. 

It is better to follow single responsability principle and each provider takes care of its own business. It is easier to test, to maintain, and to read. In this case, the `SRPanelProvider` could provide the SRPanel instance and potentially provide useful helper functions if needed for the UI components

Reply:
Ah yeah, that makes sense. I think it would simplify indeed. I think I misunderstood.

Reply:
Done. Great suggestion. Don't really having many props to drill but using a Provider/Context/useContext hook encapsulates the SRPanel functionality beautifully.